Libraries/test/Amazon.Lambda.Annotations.SourceGenerators.Tests/HttpResultsTest.cs

```diff
@@ -14,9 +14,9 @@ public class HttpResultsTest
         [Fact]
         public void OkNoBody()
         {
-            var result = HttpResults.Ok();
+            var result = () => HttpResults.Ok();
```

Comment:
This is the change mention in the description where `ValidateResult` runs across the different combinations of API Gateway responses. The unit tests were change to pass in a `Func` instead of a single instance so we didn't reuse the same `IHttpResult` for each iteration in `ValidateResult`.